src/Controls/src/Core/HandlerImpl/Picker/Picker.cs

```diff
@@ -7,6 +7,9 @@ public partial class Picker
 		{
 #if IOS
 			[PlatformConfiguration.iOSSpecific.Picker.UpdateModeProperty.PropertyName] = MapUpdateMode,
+#elif WINDOWS
+			[nameof(Picker.HorizontalOptions)] = MapHorizontalOptions,
```

Comment:
It is a Draft because I am reviewing why this is necessary in the case of the Picker in Windows.